File: app/src/main/java/com/lemuel/lemubit/fingerprinttest/presenter/TakeAttendancePresenter.java
package com.lemuel.lemubit.fingerprinttest.presenter;

import com.lemuel.lemubit.fingerprinttest.model.RealmModel;
import com.lemuel.lemubit.fingerprinttest.viewInterface.TakeAttendanceView;

public class TakeAttendancePresenter {

    public void getUserInfo(int ID, TakeAttendanceView takeAttendanceView)
    {
        RealmModel realmModel=new RealmModel();

        takeAttendanceView.onUpdateInfoTextView(realmModel.getUserInfo(ID).getName());
    }
}

File: app/src/main/java/com/lemuel/lemubit/fingerprinttest/presenter/TakeAttendancePresenter.kt
package com.lemuel.lemubit.fingerprinttest.presenter

import com.lemuel.lemubit.fingerprinttest.model.DataHelper
import com.lemuel.lemubit.fingerprinttest.viewInterface.TakeAttendanceView

object TakeAttendancePresenter {

    fun getUserInfo(fingerPrintID: Int, takeAttendanceView: TakeAttendanceView) {
        takeAttendanceView.onUpdateInfoTextView(DataHelper.getUserInfo(fingerPrintID).name)
        takeAttendanceView.onInfoGotten(DataHelper.getUserInfo(fingerPrintID).getId(), DataHelper.getUserInfo(fingerPrintID).name, DataHelper.getUserInfo(fingerPrintID).lastName)
    }

}
